engine/graphics/src/graphics.cpp

```diff
@@ -306,6 +306,58 @@ namespace dmGraphics
         }
     }
 
+    PipelineState GetDefaultPipelineState()
+    {
+        PipelineState ps;
+        ps.m_WriteColorMask           = DM_GRAPHICS_STATE_WRITE_R | DM_GRAPHICS_STATE_WRITE_G | DM_GRAPHICS_STATE_WRITE_B | DM_GRAPHICS_STATE_WRITE_A;
+        ps.m_WriteDepth               = 1;
+        ps.m_PrimtiveType             = PRIMITIVE_TRIANGLES;
+        ps.m_DepthTestEnabled         = 1;
+        ps.m_DepthTestFunc            = COMPARE_FUNC_LEQUAL;
+        ps.m_BlendEnabled             = 0;
+        ps.m_BlendSrcFactor           = BLEND_FACTOR_ZERO;
+        ps.m_BlendDstFactor           = BLEND_FACTOR_ZERO;
+        ps.m_StencilEnabled           = 0;
+        ps.m_StencilFrontOpFail       = STENCIL_OP_KEEP;
+        ps.m_StencilFrontOpDepthFail  = STENCIL_OP_KEEP;
+        ps.m_StencilFrontOpPass       = STENCIL_OP_KEEP;
+        ps.m_StencilFrontTestFunc     = COMPARE_FUNC_ALWAYS;
+        ps.m_StencilBackOpFail        = STENCIL_OP_KEEP;
+        ps.m_StencilBackOpDepthFail   = STENCIL_OP_KEEP;
+        ps.m_StencilBackOpPass        = STENCIL_OP_KEEP;
+        ps.m_StencilBackTestFunc      = COMPARE_FUNC_ALWAYS;
+        ps.m_StencilWriteMask         = 0xff;
+        ps.m_StencilCompareMask       = 0xff;
+        ps.m_StencilReference         = 0x0;
+        ps.m_CullFaceEnabled          = 0;
+        ps.m_CullFaceType             = FACE_TYPE_BACK;
+        ps.m_PolygonOffsetFillEnabled = 0;
+        return ps;
+    }
```

Comment:
These values are based on what the default OpenGL state should look like. Maybe we need to set these after the context has been created to be sure it's in sync